sdk/resourcemanager/internal/testutil/recording.go

```diff
@@ -8,95 +8,49 @@ package testutil
 
 import (
 	"fmt"
-	"net/http"
-	"strconv"
 	"testing"
 
-	"github.com/Azure/azure-sdk-for-go/sdk/azcore/policy"
 	"github.com/Azure/azure-sdk-for-go/sdk/internal/recording"
 )
 
-const recordingRandomSeedVariableName = "recordingRandomSeed"
-
-var (
-	recordingSeed int64
-)
-
-type recordingPolicy struct {
-	options recording.RecordingOptions
-	t       *testing.T
-}
-
-// Host of the test proxy.
-func (r *recordingPolicy) Host() string {
-	if r.options.UseHTTPS {
-		return "localhost:5001"
-	}
-	return "localhost:5000"
-}
+// StartRecordingProxy starts the recording proxy with the path to store recording file.
+// It should be used in the module test preparation stage only once.
+// It will return a delegate function to stop recording proxy.
+func StartRecordingProxy(pathToPackage string) func() {
```

Comment:
```suggestion
func StartProxy(pathToPackage string) func() {
```
to avoid suggesting this function is for recording mode only